### ISSUE TITLE
How to convert int64 axit to int32 for my pb model in strided_slice op?

### ISSUE THREAD

**Issue by snsun** [playbar/tfcmake#1]
Hi, 
Recentlly, I converted my pytorch model to pb using onnx-tf. I can convert successfully. But during inference, I found that in my pb model, the axis of strided_slice is int64. How can  I convert int64 to int32 axis?

https://github.com/playbar/tfcmake/blob/e5c8319827fd06146f1edcc1557f0189e50da89d/tensorflow/contrib/lite/kernels/strided_slice.cc#L151
